packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -99,7 +98,7 @@ export function useApolloUsageReport(options: ApolloUsageReportOptions = {}): Pl
   }
 
   let clientVersionFactory: StringFromRequestFn = req =>
-    req.headers.get('apollographql-client-version') || yoga.version;
```

Comment:
Isn't it nicer to have some information about the sender even if it is not dynamic because otherwise the information becomes anonymous?

Reply:
I don't think so. 🙂 It will always be an anonymous call, even if we add some default value here. Additionally, if someone sends only "client name", then some unrelated version value **should not be added here**. As I mentioned above, it would be a mess. 🙂
It is normal behavior for Apollo Studio that the client name or version does not always appear in the trace and these metrics can be still properly filtered (as "unidentified client"). Even the official implementation of this plugin does not assign any default values. Check it out [here](https://github.com/apollographql/apollo-server/blob/90832fd2b1e4be9f3fb4baf704f348f8975fd0ea/packages/server/src/plugin/usageReporting/plugin.ts#L504).